### PR TITLE
Refactor build scripts

### PR DESCRIPTION
**Pull request by IlyaKhD**

### CODE REVIEW

File: .github/workflows/packages_publishing.yml [hunk 1]
```diff
@@ -50,17 +50,17 @@ jobs:
...
+        run: npm run ts-node build/set-timestamp-version
 
       - name: Build npm packages
         env:
           BUILD_INTERNAL_PACKAGE: true
         run: npm run all:build
 
       - name: Build artifacts package
-        run: npm run ts-script build/make-artifacts-package
+        run: npm run ts-node build/make-artifacts-package
```
Comment (pomahtri) [inline .github/workflows/packages_publishing.yml]: Why not just `npx ts-node ...`? The behavior should be the same, npx calls bin packages from local node_modules if they exist

Comment (IlyaKhD) [reply .github/workflows/packages_publishing.yml]: You're right, no more need in script (I used it in local experiments)

Fixed, thanks!

Comment (pomahtri) [reply .github/workflows/packages_publishing.yml]: Remove it from package.json `scripts` field too please

Comment (IlyaKhD) [reply .github/workflows/packages_publishing.yml]: done

**Review by alexslavr — APPROVED** [review]

**Pull request merged**